### PR TITLE
allow for escaping the grid, dev portal 

### PR DESCRIPTION
**Pull request by jkreft-usgs**
Before making a pull request
----------------------------

- [ ] Update the changelog appropriately

Title
-----------
Add a developer portal

Description
-----------
The goal here is to try out using rapidoc to create a developer portal

---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: themes/wdfn_theme/src/styles/main.scss [hunk 1]
```diff
@@ -1,4 +1,30 @@
-@import '../../node_modules/wdfn-viz/dist/sass/wdfnviz';
+@import '../../node_modules/wdfn-viz/dist/sass/theme';
+@import '../../node_modules/wdfn-viz/dist/sass/uswds';
+
+$usgs-blue: #00264c;
+
+@import '../../node_modules/wdfn-viz/dist/sass/partials/header';
+@import '../../node_modules/wdfn-viz/dist/sass/partials/footer';
```
Comment (mbucknell) [inline themes/wdfn_theme/src/styles/main.scss:7]: Why was this necessary?  I pulled the code down and restored the original single import and I didn't see any differences on the roadmap page, so I would like the original single import restored.

Comment (jkreft-usgs) [reply themes/wdfn_theme/src/styles/main.scss:7]: the roadmap page isn't the problem.  Try one of the rapidocs pages off of api-docs
